static/app/components/events/interfaces/debugMeta/debugImageDetails/index.tsx

```diff
@@ -260,14 +259,12 @@ export function DebugImageDetails({
     displayReprocessEventAction(organization.features, event) &&
     !!onReprocessEvent;
 
-  if (isLoading) {
-    return <LoadingIndicator />;
-  }
-
   if (isError) {
     return <LoadingError />;
   }
 
+  const shouldShowLoadingIndicator = isLoading && hasUploadedDebugFiles;
```

Comment:
This also fixes an infinite loading spinner when there are no candidates. Removed the full screen loading spinner above because one is shown below in a more specific manner.